benchmarks/bench.rs

```diff
@@ -99,7 +99,7 @@ fn bench_rustpy_nbody(b: &mut test::Bencher) {
     };
 
     b.iter(|| {
-        let scope = vm.ctx.new_scope();
+        let scope = vm.ctx.new_scope(&vm);
```

Comment:
I don't like the method on context taking a ```VirtualMachine```. Given that this now needs vm then I think the method should move to the ```VirtualMachine```.